docs/userGuide/tweakingThePageStructure.md

```diff
@@ -306,4 +306,25 @@ In this case `myHead.md` will override the `chapterLayout/head.md`.
 
 **The layout named `default` (if any) is automatically applied to every single page.** When you `init` a MarkBind site, MarkBind also generates an empty `default` layout.
 
+### Custom JavaScript in `scripts.js`
```

Comment:
Since this section focuses on specifically on the `afterSetup` function, we can rename the heading to something that fits this section better.

How about "Attaching Event Listeners in `scripts.js`"